packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-point.tsx

```diff
@@ -8,6 +8,11 @@ import type {vec} from "mafs";
 
 type Props = {
     point: vec.Vector2;
+    /**
+     * Represents where this point stands in the overall point order.
+     * This is used to provide screen readers with context about the point.
+     */
+    currentPointOrder: number;
```

Comment:
I'd prefer the name `sequenceNumber` for this prop. See: https://en.wikipedia.org/wiki/Sequence_number. `current` is confusing to me because props are always current.

Maybe the doc comment should also mention that this number is 1-based, and that the numbering is supposed to restart from 1 for each interactive figure in the graph. But I could also see an argument for not documenting any of that, since the MovablePoint component can't control how people use it. All we really need to say in this comment is that the number will be presented to screenreader users as-is.